eng/packaging.targets

```diff
@@ -42,4 +54,76 @@
     <MSBuild Projects="$(PkgProjPath)"
              Targets="Build" />
   </Target>
+    
+  <Target Name="AddRefAssemblies" Condition="'$(IncludeRefInPackage)' == 'true' and '$(TargetsAnyOS)' == 'true' and '@(ResolvedMatchingContract)' != ''"> 
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="@(ResolvedMatchingContract)">
+        <RefAssemblyOutput>true</RefAssemblyOutput>
+        <PackagePath>ref/$([System.String]::new('%(ResolvedMatchingContract.SetTargetFramework)').Substring(16))</PackagePath>
```

Comment:
Is 16 some magic number that others or I should know of? :D

Reply:
hahaha its the property value is length of "TargetFramework="

Reply:
Please add a comment to make that clear.